pennylane/templates/state_preparations/state_prep_mps.py

```diff
@@ -15,24 +15,124 @@
 Contains the MPSPrep template.
 """
 
+import numpy as np
+
 import pennylane as qml
 from pennylane.operation import Operation
 from pennylane.wires import Wires
 
 
+def right_canonicalize_mps(mps):
+    r"""Transform a matrix product state into its right-canonical form.
+
+    Args:
+        mps (list[TensorLike]): List of tensors representing the MPS.
```

Comment:
It is very hard to see what the correct shape of the MPS should be. There are some explanation in MPSPrep under [Usage Details](https://xanaduai-pennylane--6896.com.readthedocs.build/en/6896/code/api/pennylane.MPSPrep.html#usage-details) which could be copied here as well. @DSGuala any thoughts or suggestions here?